action/protocol/staking/validations.go

```diff
@@ -39,13 +39,10 @@ func (p *Protocol) validateCreateStake(ctx context.Context, act *action.CreateSt
 	if act.Amount().Cmp(p.config.MinStakeAmount) == -1 {
 		return errors.Wrap(ErrInvalidAmount, "stake amount is less than the minimum requirement")
 	}
-	if act.GasPrice().Sign() < 0 {
-		return errors.Wrap(action.ErrGasPrice, "negative value")
-	}
```

Comment:
can remove same check in other actions like Unstake, Withdraw, CandRegister, etc.